django_short_urls/auth.py

```diff
@@ -11,6 +11,20 @@
 from django.http import HttpResponse
 
 
+def _safe_base64_decode(encoded):
+    """Method that base64 decodes a string, while gracefully handing insufficient padding.
```

Comment:
Yes, the client is most likely faulty in those cases, but based on the comments on the StackOverflow page, it seemed to occur in some cases (seems sometimes reasonable clients, such as "PHP", simply don't add the padding).